hail/www/style.css

```diff
@@ -232,36 +229,34 @@ table td {
   vertical-align: top;
 }
 
-.author {
-  font-size: 1.2em;
-  text-align: center;
+@media only screen and (max-width: 768px) {
+  html {
+    font-size: 87.5%; /*14px* w/browser default 16px*/
```

Comment:
allows us to control fonts using rem, while also not overriding the browser's font size setting